core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -324,8 +324,7 @@ private void doExecute0(ClientRequestContext ctx, HttpRequestDuplicator rootReqD
             response = executeWithFallback(unwrap(), derivedCtx,
                                            (context, cause) -> HttpResponse.ofFailure(cause));
         }
-
-        final RetryConfig<HttpResponse> config = mapping().get(ctx, duplicateReq);
+        final RetryConfig<HttpResponse> config = mappedRetryConfig(ctx);
```

Comment:
Could you also fix `RetryingRpcClient`, the counterpart?
https://github.com/line/armeria/blob/aa3c6e1acfd41c8ab4f15f2eff2051a87bc8cc18/core/src/main/java/com/linecorp/armeria/client/retry/RetryingRpcClient.java#L189